### ISSUE TITLE
FeatureR: setSelection(selection, options)

### ISSUE THREAD

**Issue by rexbut** [hypery2k/ngx-vis#106]
## What did you do?

Tried calling the function setSelection(selection, options) after the event "click" to select nodes and edges.

## What did you expect to happen?

Sets the selection, wich must be an object like this:
```
{
  nodes: [Array of nodeIds],
  edges: [Array of edgeIds]
}
```
http://visjs.org/docs/network/index.html?keywords=setSelection

## What happened instead?

Method is not implemented

## Your Environment

**Angular version:** 6.0.1
**NodeJS and NPM version:** 6.1.0

**Comment by hypery2k** [comment]
currently having much time for this project, but any PR is welcome

**Comment by hypery2k** [comment]
published with v1.0.1
